src/fedora/33/helix/amd64/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-33
```

Comment:
this tag won't exist unless you give it a dockerfile

Reply:
(That other Dockerfile will also need an entry in the manifest.json)

Reply:
I copied this from https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/master/src/fedora/32/helix/amd64/Dockerfile#L1 what should I put here?

Reply:
That tag in the Fedora 32 DockerFile is a reference to https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/master/src/fedora/32/amd64/Dockerfile .  During building this will get generated and tagged as such; it's a form of inheritance.

To be clear, you just need to add src/fedora/33/amd64/Dockerfile and its corresponding manifest entry.

Reply:
Alrighty cool, I added copies of the non helix dockerfile/manifest too